files/vt

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Provides a shorthand for fab vagrant test_module:module_name
+# Allows use of bash auto-completion for deploying modules to vagrant vm
+# Does not require removal of trailing / characters
+# Automates movement of directories to redcap_deployment/root
+
+# Replace with the path to your redcap_deployment directory
+redcap_deployment_root_dir="/Users/kyle.chesney/projects/redcap_deployment"
```

Comment:
Any objections to setting the path here in the file?
I considered loading it in from a separate config file but then it would have to be brought along to `/usr/local/bin`, I also considered auto-generating the config with `find` but that seemed like over-engineering